dotnet/src/SemanticKernel.Core/Text/TextChunker.cs

```diff
@@ -84,8 +103,19 @@ public static List<string> SplitMarkDownLines(string text, int maxTokensPerLine,
     /// <param name="chunkHeader">Text to be prepended to each individual chunk.</param>
     /// <param name="tokenCounter">Function to count tokens in a string. If not supplied, the default counter will be used.</param>
     /// <returns>List of paragraphs.</returns>
-    public static List<string> SplitPlainTextParagraphs(IEnumerable<string> lines, int maxTokensPerParagraph, int overlapTokens = 0, string? chunkHeader = null, TokenCounter? tokenCounter = null) =>
-        InternalSplitTextParagraphs(lines, maxTokensPerParagraph, overlapTokens, chunkHeader, static (text, maxTokens, tokenCounter) => InternalSplitLines(text, maxTokens, trim: false, s_plaintextSplitOptions, tokenCounter), tokenCounter);
+    public static List<string> SplitPlainTextParagraphs(
+    IEnumerable<string> lines,
+    int maxTokensPerParagraph,
+    int overlapTokens = 0,
+    string? chunkHeader = null,
+    TokenCounter? tokenCounter = null) =>
+    InternalSplitTextParagraphs(
+        lines.Select(line => line.Replace("\r\n", "\n").Replace('\r', '\n')),
```

Comment:
A slightly faster way to normalize with the assumption that lone `\r` are no longer valid line endings on their own. This was the case for older Unix platforms, but virtually no OS supports/uses lone `\r` as a laine ending.
```suggestion
        lines.Select(line => line.Replace("\r", ""),
```